c/cruise-control/redhat_ubi8/Dockerfile

```diff
@@ -1,5 +1,3 @@
-FROM registry.access.redhat.com/ubi8/ubi
-RUN yum install vim java-1.8.0-openjdk-devel maven git -y
-env JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk
-env JRE_HOME=/usr/lib/jvm/java-1.8.0-openjdk/jre
+FROM adoptopenjdk/openjdk8-openj9:ubi 
```

Comment:
Hi Amir,
Why not `adoptopenjdk:8-jdk-openj9` instead?

adoptopenjdk:8-jdk-openj9 = [Official Containers Images ](https://hub.docker.com/_/adoptopenjdk?tab=tags&page=1&ordering=last_updated)

adoptopenjdk/openjdk8-openj9:ubi  = [Unofficial Containers Images](https://hub.docker.com/r/adoptopenjdk/openjdk8-openj9/)

Is it because [Official Containers Images ](https://hub.docker.com/_/adoptopenjdk?tab=tags&page=1&ordering=last_updated) are not UBI based?

Reply:
Yes, it is an ubuntu-based image. But, to be honest, I rather use ubuntu :)